data-ingestion-service/src/main/java/gov/cdc/dataingestion/camel/routes/ElrProcessStatusComponent.java

```diff
@@ -0,0 +1,73 @@
+package gov.cdc.dataingestion.camel.routes;
+
+import gov.cdc.dataingestion.reportstatus.model.EdxActivityLogStatus;
+import gov.cdc.dataingestion.reportstatus.model.MessageStatus;
+import gov.cdc.dataingestion.reportstatus.service.ReportStatusService;
+import org.apache.camel.Handler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class ElrProcessStatusComponent {
+    private static Logger logger = LoggerFactory.getLogger(ElrProcessStatusComponent.class);
+    private ReportStatusService reportStatusService;
+
+    private static final String SUCCESS = "Success";
+    private static final String FAILURE = "Failure";
+    private static final String MSG_STATUS_FAILED = "FAILED";
+    private static final String ELR_ID = "ELR Id";
+    private static final String STATUS_VALIDATION_ERROR ="Status: Validation Error";
+    private static final String CREATED_ON ="Created On";
+    private static final String ERROR_MESSAGE ="Error Message";
+
+    @Autowired
+    public ElrProcessStatusComponent(ReportStatusService reportStatusService) {
+        this.reportStatusService = reportStatusService;
+    }
+
+    @Handler
+    public String process(String body){
+        String status = "";
+            logger.debug("File name and Elr Id:{}", body);
+            if (body != null && !body.trim().isEmpty()) {
+                //body = "HL7file-sftpstatus1.txt:7DAC34BD-B011-469A-BF27-25904370E9E3";//NOSONAR
+                String[] fileNameElr = body.split(":");
+                String elrId = fileNameElr[1].trim();
+                MessageStatus messageStatus = reportStatusService.getMessageStatus(elrId);
+                if (messageStatus.getNbsInfo().getNbsInterfaceStatus() != null && messageStatus.getNbsInfo().getNbsInterfaceStatus().equals(SUCCESS)) {
+                    status = SUCCESS;
```

Comment:
Should we also append ELR ID when then message is success?

Reply:
Anil said a 'Success' message is enough when the record has been processed successfully.